person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    true if @age >= 18
+  end
```

Comment:
- [ ] **Optional:** You can consider restructuring this code as follows.

```
  def of_age?
    @age >= 18
  end
```

This is because a conditional statement always returns Boolean `true` or `false`.